### ISSUE TITLE
(Android) Switch Mapbox account

### ISSUE THREAD

**Issue by fleuverouge** [mapbox/mapbox-gl-native#9770]
**Platform: Android**
**Mapbox SDK version: 5.1.1.2**

Hi, 
I'd like to switch between Mapbox accounts in my Android app. However, as far as I can see in [Mapbox class](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/Mapbox.java#L51), the INSTANCE can only be created once and access key is not able to be changed. I'll be very appreciate if the sdk supports multiple accounts in the next release(s)!

**Comment by tobrun** [comment]
@fleuverouge thank you for reaching out, first blocker for this is https://github.com/mapbox/mapbox-gl-native/issues/9740. Work for that is happening in https://github.com/mapbox/mapbox-java/pull/537. Once that is unblocked we can look into adding https://github.com/mapbox/mapbox-gl-native/issues/7584. Closing this for now as a duplicate of #7584. Feel free to report any other issues you are facing!
